nodejs/src/stream_manager.rs

```diff
@@ -544,7 +544,7 @@ impl StreamManager {
             scaling_policy.scaling.clone(),
             retention_policy.retention.clone(),
             match tags.len() {
-                l if l == 0 => None,
+                0 => None,
```

Comment:
Updated this line as per the suggestion provided in the failed build logs.
![image](https://github.com/pravega/pravega-client-rust/assets/122247323/7cd4c202-4a87-41b5-824f-eba140d3850b)